orttraining/orttraining/core/framework/gradient_graph_builder.h

```diff
@@ -40,7 +40,8 @@ static std::unordered_map<std::string, std::unordered_set<size_t>>
         {"Scatter", {1}},
         {"OneHot", {0, 1, 2}},
         {"Where", {0}},
-        {"Range", {0, 1, 2}}};
+        {"Range", {0, 1, 2}},
+        {"BroadcastGradientArgs", {0, 1}}};
```

Comment:
FYI, this is not strictly needed. as this op shouldn't be used in the inference graph.

Reply:
might still be okay to leave it in just in case this situation is encountered?